src/table/styles.scss

```diff
@@ -79,6 +77,10 @@
   &.has-footer {
     padding-bottom: 0px;
   }
+  &:not(.has-header) {
```

Comment:
the issue says it only happens in embedded tables, why do we need to remove the border from every table?

Reply:
It actually happens to not only embedded. But in general, when there is already a header. For example `container`:
<img width="695" alt="Screenshot 2022-11-28 at 11 14 54" src="https://user-images.githubusercontent.com/98534165/204252265-e3032194-6d42-4457-9184-c1275ae2b0c7.png">